### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by marilirulita**
### Set up associations between classes and objects.
**In this Branch I create some new classes:**
- **One new class Classroom and a relationship has-many/belongs-to with students**
- **Two classes, book, and rental to create a relationship many-to-many between book and person using rental as a connection**

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```
Comment (gemmen29) [inline person.rb:15]: - [ ] Kindly create a method that creates a new rental object of class Rental and passes `date`, `book`, and `person` itself as a parameter to achieve the `has-many` association. You may learn more about both ways association from [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :tittle, :author
+  attr_reader :rentals
+
+  def initialize(tittle, author)
+    @tittle = tittle
+    @author = author
+    @rentals = []
+  end
```
Comment (gemmen29) [inline book.rb:9]: - [ ] Kindly create a method that creates a new rental object of class Rental and passes `date`, `person`, and `book` itself as a parameter to achieve the `has-many` association. You may learn more about both ways association from [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @marilirulita, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 2 above)
Comment (marilirulita) [reply book.rb:9]: thanks for the message @gemmen29 It's done!

**Review by andersonlebon — APPROVED** [review]
## Hi @marilirulita 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**